src/inmanta/protocol/openapi/converter.py

```diff
@@ -405,6 +405,7 @@ def handle_method(self, url_method: UrlMethod, path: str) -> Operation:
             parameters=(parameters if len(parameters) else None),
             summary=url_method.short_method_description,
             description=url_method.long_method_description,
+            tags=[url_method.endpoint.__class__.__name__],
```

Comment:
I think we should be able to use the name of the slice. Not 100% sure but this could work:
```suggestion
            tags=[url_method.endpoint._name],
```